docs/cas-server-documentation/protocol/CAS-Protocol.md

```diff
@@ -40,6 +40,9 @@ The version `2.0` protocol specification is available at [CAS-Protocol-Specifica
 ## Proxy web flow diagram
 One of the most powerful feature of the CAS protocol is the ability for a CAS service to act as a proxy for another CAS service, transmitting the user identity.
 
+<div class="alert alert-warning">Since version 6.1, if CAS is configured to return the proxy-granting ticket id directly in the validation response,
```

Comment:
Remove this please. 

The ability to return a PGT in the response is a CAS-server implementation feature; not a protocol clause. Amending the protocol would require consensus from the PMC, and would also impact all other implementations of the CAS protocol.

Reply:
Will do